tools/plugins/com.liferay.ide.core/.classpath

```diff
@@ -1,15 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
-		<attributes>
-			<attribute name="module" value="true"/>
-		</attributes>
-	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="src" path="src-external"/>
+	<classpathentry kind="src" path="src/"/>
+	<classpathentry kind="src" path="src-external/"/>
 	<classpathentry exported="true" kind="lib" path="lib/commons-configuration-1.6.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/freemarker-2.3.17-patched.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/commons-io-2.6.jar"/>
```

Comment:
Lets use the commons-io that is running as a bundle inside of eclipse.  Also you ahve commons-io as another embedded dependency in the server.core plugin, i think we can remove them both and use the bundled installed and running in eclipse.